kpi/tests/api/v1/test_api_users.py

```diff
@@ -1,7 +1,46 @@
 # coding: utf-8
+from django.contrib.auth.models import User
+from rest_framework.reverse import reverse
+
 from kpi.tests.api.v2 import test_api_users
 
 
 class UserListTests(test_api_users.UserListTests):
 
     URL_NAMESPACE = None
+
+    def test_current_user_extra_details_kludges(self):
+        self.client.login(username='someuser', password='someuser')
+        user = User.objects.get(username='someuser')
+        xtradata = user.extra_details.data
+        assert xtradata == {}
+
+        # `primarySector` should be renamed to `sector`
+        xtradata['primarySector'] = 'camelCase Administration'
+        user.extra_details.save()
+        response = self.client.get(reverse('currentuser-detail'))
```

Comment:
Would it better to call `reverse(self._get_endpoint('currentuser-detail'))` even if `CurrentUserViewSet` does not exist in `v2` yet? 
When (if?) we port the viewset to a newer version of API, we could copy the tests and they should keep working.
What do you think?

Reply:
fine